frontend/src/components/horizontalPodAutoscaler/__snapshots__/HPADetails.Error.stories.storyshot

```diff
@@ -79,7 +79,7 @@
               <p
                 class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter css-9ldkza-MuiTypography-root"
               >
-                Error: Unreachable
+                TypeError: Failed to fetch
```

Comment:
Is this expected?

Reply:
Yes, "Unreachable" message was hard coded, now it has an actual original error message